src/common/constants.ts

```diff
@@ -1,10 +1,9 @@
 export const SIDE_DRAWER_WIDTH = 233;
 export const DATE_FORMAT = 'DD/MM/YY';
 export const TIME_FORMAT = 'HH:mm';
-export const REACT_APP_GLIFIC_REGISTRATION_API =
-  process.env.REACT_APP_GLIFIC_API + '/v1/registration';
+export const REACT_APP_GLIFIC_REGISTRATION_API = process.env.REACT_APP_GLIFIC_API ? process.env.REACT_APP_GLIFIC_API + '/v1/registration' : 'https://api.glificuat.coloredcow.com/api/v1/registration';
 export const REACT_APP_GLIFIC_AUTHENTICATION_API =
-  process.env.REACT_APP_GLIFIC_API + '/v1/registration/send-otp';
-export const USER_SESSION = process.env.REACT_APP_GLIFIC_API + '/v1/session';
-export const RESET_PASSWORD = process.env.REACT_APP_GLIFIC_API + '/v1/registration/reset-password';
+  process.env.REACT_APP_GLIFIC_API ?  process.env.REACT_APP_GLIFIC_API + '/v1/registration/send-otp': 'https://api.glificuat.coloredcow.com/api/v1/registration/send-otp';
```

Comment:
We can't hardcode URLs here.

Reply:
I agree @kurund that was just a workaround for testing CD. I've figured out about the env variables now and reverted these changes now.